codebasin/util.py

```diff
@@ -141,6 +113,90 @@ def valid_path(path):
     return valid
 
 
+def _validate_json(json_object: str, schema_name: str) -> bool:
```

Comment:
This is overly pedantic and doesn't matter because it's not so public facing, but this function is JSON centric but it's not actually; `config.py` feeds the output of a YAML file into this, so you might want to rename the function more generically to be like `_validate_config`, and pass a `config: dict[str, Any]`

Reply:
I can see where you're coming from.

I'm hoping that we'll be able to remove the YAML configuration files completely in 2.0.0...  Do you think we should rename the function for now (to reflect that it's occasionally used for JSON and YAML), but change the name back when we remove the YAML?

Reply:
Ok, I decided to split the difference: see 7256a84cb7719054ce09c18ae506f67380b6e7c5.

We can keep the existing `_validate_json` function, and remove the new `_validate_yaml` function when we no longer need it.